tests/openwisp2/settings.py

```diff
@@ -31,7 +31,10 @@
     'OPTIONS': {'udp_writes': True, 'udp_port': 8089},
```

Comment:
Do not default to UDP as that will cause issues with new developers trying this module out. Default on TCP and enable UDP in the tests.